README.md

```diff
@@ -16,7 +16,7 @@ Documentation
 -------------
 
 Documentation for how to run and extend Sawtooth is available here:
-https://sawtooth.hyperledger.org/docs/seth/
+https://sawtooth.hyperledger.org/docs/seth/nightly/master/introduction.html
```

Comment:
This will become inaccurate when we do a release.  Maybe we should create the 'lastest' link pointing to nighty until we start doing releases. (And maybe we start doing Seth releases soon.). Thoughts?

Reply:
True, but I think if we keep it pointed at the nightly docs until then it will be more helpful...

Reply:
This URL should work now: https://sawtooth.hyperledger.org/docs/seth/releases/latest/introduction.html

I links to nightly master until we do a release.

Reply:
Changed now